src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -2751,3 +2751,41 @@ export function getMolecularProfileSamplesSet(
         {}
     );
 }
+
+export function getMolecularProfilesByAlterationType(
```

Comment:
i think the role of dataTypePreference complicates this function.  Like, we're not just getting the profiles for a particular alteration type, as the name suggests, but also applying some other kind of filtering

Reply:
how about `getFilteredMolecularProfilesByAlterationType`?